config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml

```diff
@@ -195,29 +199,7 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:v20181005-fd9cfb8b0-master
 
 - interval: 30m
-  name: ci-kubernetes-e2e-gci-gce-etcd3
```

Comment:
why the removal of this job?

Reply:
we ditched etcd2, now this one is the same as default gci-gce job /shrug